### PR TITLE
ad_build_changes, look at biller-build.txt

### PR DESCRIPTION
**Pull request by abhidwiIBM**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -106,11 +106,13 @@
 			<artifactId>db2jcc4</artifactId>
 			<version>10.1</version>
 		</dependency>
+		<!--
```
Comment (vivekveera) [inline pom.xml]: why do we still need db2jcc.jar in bin folder when pom dependency for this jar is commented out ?

Comment (abhidwiIBM) [reply pom.xml]: dependency is commented only for db2jcc and not for db2jcc4, db2jcc4.jar will be installed.

Comment (abhidwiIBM) [reply pom.xml]: we don't need db2jcc.jar as higher version db2jcc4.jar has already been added.

Comment (abhidwiIBM) [reply pom.xml]: It will be good If you can remove db2jcc.jar from db2-connector folder.

Comment (vivekveera) [reply pom.xml]: yes..was asking about db2jcc only not db2jcc4. so from which folder we need to drop db2jcc.jar ? was it not bin folder ?

Comment (vivekveera) [reply pom.xml]: remove maven dependency for db2jcc.jar as we don't need it anymore.

Comment (abhidwiIBM) [reply pom.xml]: No jar is added in bin folder, db2jcc4.jar has been added in db2-connector folder and to add it to maven repository, we need to run the command mentioned in biller-build.txt only once on our local.
Dependency is commented in pom.xml, if you want you can remove it.

**Pull request merged**